test/honeysql/core_test.clj

```diff
@@ -15,6 +15,7 @@
                (join :draq [:= :f.b :draq.x])
                (left-join [:clod :c] [:= :f.a :c.d])
                (right-join :bock [:= :bock.z :c.e])
+               (full-join :beck [:= :beck.x :c.y])
```

Comment:
I didn't see any tests for merge join logic, so I wasn't sure where the best place would be to add it.